routers/likes.py

```
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+@router.post("/{postId}/likes")
+def do_like(postId: int):
```

Ruff (0.14.11)

Unused function argument: `postId`

(ARG001)